inspirehep/migrator/api.py

```diff
@@ -28,25 +28,27 @@ def continuous_migration():
     if redis_url is None:
         redis_url = current_app.config.get("CACHE_REDIS_URL")
 
-    r = StrictRedis.from_url(redis_url)
-    lock = Lock(r, "continuous_migration", expire=120, auto_renewal=True)
+    redis = StrictRedis.from_url(redis_url)
```

Comment:
👍 for renaming ``r``. ``redis`` can be easily confused with the ``redis`` module IMHO

Reply:
`redis` -> `redis_cli` for redis client object